dashboard/app/models/pd/application/teacher1920_application.rb

```diff
@@ -330,6 +330,28 @@ def friendly_scholarship_status
       end
     end
 
+    def allow_sending_principal_email
```

Comment:
Nit: This method name sounds like a command, not a query.  Since it returns a bool, adding the question mark `allow_sending_principal_email?` might clear things up.

Reply:
It's a good point, and I had actually written it like that briefly, but only changed back because none of the other getters that we serialize in the model (e.g. `meets_criteria`, `meets_scholarship_criteria`) used the `?`...

Reply:
Still, it does read better, so now added.